jira/client.py

```diff
@@ -92,7 +92,8 @@ def emit(self, record):
 #    "which means that there is a big change of having problems. " \
 #    "Possible workaround http://stackoverflow.com/a/17628350/99834" % encoding)
 
-logging.getLogger('jira').addHandler(NullHandler())
+logger = logging.getLogger('jira')
```

Comment:
`'jira'` logger name was the original one (was there before this PR). Using `__name__` for a logger name is not correct here, because `__name__ = 'client'`, which does not describe the library logger properly. However, naming it after `__package__` would be a more correct way.

FYI: Werkzeug uses harcoded logger name: https://github.com/pallets/werkzeug/blob/423a356d4a49a50179a2be7ba131570dbc607a34/werkzeug/_internal.py#L80